### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.DS_Store
```
Comment (James-Garden) [inline src/.gitignore]: Can we move all these files to a directory called 'frontend'?

File: src/src/databaseRecipeCode.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
+CREATE DATABASE recipes;
```
Comment (James-Garden) [inline src/src/databaseRecipeCode.sql]: Why do we have a SQL script in a front-end PR?

File: src/src/assets/AccountSettings.png
Comment (James-Garden) [inline src/src/assets/AccountSettings.png:1]: Besides the logo, I would probably just put these as text, not as images. If we want to change anything, we'll have to make a whole new image.

**Review by James-Garden — CHANGES_REQUESTED** [review]
Could you put some screenshots of what these pages look like?

File: src/src/databaseRecipeCode.sql (hunk 2 above)
Comment (ghost) [reply src/src/databaseRecipeCode.sql]: i downloaded the SQL file from github, i just try to everything on the database can show with my front end.

File: src/src/assets/AccountSettings.png
Comment (ghost) [reply src/src/assets/AccountSettings.png:1]: sure, i will fix it, just use the current version then i will update later

File: src/.gitignore (hunk 1 above)
Comment (ghost) [reply src/.gitignore]: U mean, change the folder SRC to Frontend?

Comment (James-Garden) [reply src/.gitignore]: No, I just mean create a folder called 'frontend' at the root, and move all of these files including the src folder into it.

**Review by James-Garden — APPROVED** [review]
👍 Looks good to me

Just to nitpick, folder names are usually lowercase, so I would just put 'frontend' instead of 'Front-End'

**Pull request merged**